### PR TITLE
Fixed link

### PR DESCRIPTION
**Pull request by justanotheranonymoususer**

### CODE REVIEW

**Comment by PRMerger19** [comment]
@justanotheranonymoususer : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/interlockedapi/nf-interlockedapi-interlockedpushentryslist.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 UID: NF:interlockedapi.InterlockedPushEntrySList
 title: InterlockedPushEntrySList function (interlockedapi.h)
-description: Inserts an item at the front of a singly linked list. Access to the list is synchronized on a multiprocessor system.helpviewer_keywords: ["InterlockedPushEntrySList","InterlockedPushEntrySList function","_win32_interlockedpushentryslist","base.interlockedpushentryslist","interlockedapi/InterlockedPushEntrySList","winbase/InterlockedPushEntrySList"]
+description: Inserts an item at the front of a singly linked list. Access to the list is synchronized on a multiprocessor system.
+helpviewer_keywords: ["InterlockedPushEntrySList","InterlockedPushEntrySList function","_win32_interlockedpushentryslist","base.interlockedpushentryslist","interlockedapi/InterlockedPushEntrySList","winbase/InterlockedPushEntrySList"]
```
Comment (justanotheranonymoususer) [inline sdk-api-src/content/interlockedapi/nf-interlockedapi-interlockedpushentryslist.md:5]: That was modified by the GitHub editor, I didn't do anything

Comment (ALaborate) [reply sdk-api-src/content/interlockedapi/nf-interlockedapi-interlockedpushentryslist.md:5]: That behaviour seems strange to me. Just decided to mention that I had the same issue in #247

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Pull request merged**